website/public/index.ejs.html

```diff
@@ -13,6 +13,7 @@
         border-collapse: collapse;
         width: 100%;
       }
+
```

Comment:
nit: all these formatting changes make this diff difficult to read.  Please revert formatting changes or put them in a separate commit

Reply:
Must fix: I agree. Why are there so many formatting changes? Undo them.

Reply:
Sorry, I initially didn't make those changes. After I finished the working on the issue I followed the [contributing guidelines](https://github.com/quick-lint/quick-lint-js/blob/1d71c38fb3ac3483ef97c65242466d9250763fd4/docs/CONTRIBUTING.md) and ran 
'yarn install'
'yarn fmt'
and it adjusted the file from there. 

So to be clear, I should undo the changes from the 'yarn fmt' command and resubmit?

Reply:
That's odd. When I run `yarn fmt` on master, I get no changes at all. Are you sure something else (like your editor) didn't cause those changes?

Reply:
Ok, I _think_ I have this sorted out. I apologize for the craziness, I suppose maybe there is something in my editor that formats things differently. I'm still learning so it's been a struggle. Thanks for the guidance. Does this appear to be better?